crates/orderbook/openapi.yml

```diff
@@ -965,3 +989,18 @@ components:
             the fee after this expiration date. Encoded as ISO 8601 UTC.
           type: string
           example: "1985-03-10T18:35:18.814523Z"
+    SolverCompetitionResponse:
```

Comment:
Comment by Filius on Discord:

It would be great if this object could contain some additional information:
1. The objective value provided in terms of its subcomponents (surplus, fees, cost). I've seen some solutions which have the same surplus and fees, but less costs. This would make it easy for the solver developer to quickly understand the winner is winning due to better execution. 
2. The clearing prices achieved by the solver. That would help solvers benchmark their liquidity sources with those of winning solvers. 
3. The IDs of the orders filled by the solver settlement.

Reply:
All of this is technically already there in the calldata but it is not easy to extract. So we could include this duplicate information directly to make it easier to consume.

Reply:
Agree that the `objective` can be something like:
```js
{
  // ...
  objective: {
    total: "1000.0",
    surplus: "1100.6",
    fees: "50.5",
    cost: "-151.1",
  },
  // ...
}
```